cumulus/pallets/xcmp-queue/src/mock.rs

```diff
@@ -337,12 +337,12 @@ pub(crate) fn mk_page() -> Vec<u8> {
 	page
 }
 
-pub(crate) fn v2_xcm() -> VersionedXcm<()> {
-	let instr = xcm::v2::Instruction::<()>::ClearOrigin;
-	VersionedXcm::V2(xcm::v2::Xcm::<()>(vec![instr; 3]))
-}
-
 pub(crate) fn v3_xcm() -> VersionedXcm<()> {
 	let instr = xcm::v3::Instruction::<()>::Trap(1);
 	VersionedXcm::V3(xcm::v3::Xcm::<()>(vec![instr; 3]))
 }
+
+pub(crate) fn v4_xcm() -> VersionedXcm<()> {
```

Comment:
I would maybe refactor this tests to avoid using concrete versions, something like this should be enough:

```
pub(crate) fn versioned_xcm(v: XcmVersion) -> VersionedXcm<()> {
	let instr = Instruction::<()>::Trap(1);
    VersionedXcm::from(Xcm(vec![instr; 3])).into_version(v).expect("...")
}
```

and also use:
```
    let newer_xcm_version = xcm::prelude::XCM_VERSION;
    let older_xcm_version = newer_xcm_version - 1;
```

so when adding new XCMs we won't need to come back here again